packages/dd-trace/test/profiling/tagger.spec.js

```diff
@@ -80,13 +100,6 @@ describe('tagger', () => {
     expect(parsed).to.deep.equal({})
   })
 
-  it('should ignore empty values in strings', () => {
```

Comment:
What happened with this test?

Reply:
I thought it's a duplicate of the test right before it

Reply:
Oh, you're right!